src/zip_files.ts

```diff
@@ -0,0 +1,112 @@
+import { zip } from "d3";
+import { parse, ParseResult } from "papaparse";
+import { fetchRegionalTrendsData, RegionalTrendLine } from "./data";
+import { mapData, regionalTrends } from "./stores";
+
+let rTL: RegionalTrendLine[];
+
```

Comment:
please rename this file zip_data.ts for consistency, or just add the functions to data.ts.